apps/authentication-service/cypress/e2e/clients.cy.ts

```diff
@@ -69,16 +87,48 @@ describe("Client API", () => {
         ContentType: "application/json",
         Authorization: `Bearer ${accessToken}`,
       },
-    }).then((response: Cypress.Response<{ body: Keys }>) => {
-      const { body } = response;
-      newClientId = (body as unknown as Keys).clientId;
-      newClientSecret = (response.body as unknown as Keys).clientSecret;
+    })
+      .then((response: Cypress.Response<{ body: Keys }>) => {
+        const { body } = response;
+        newClientId = (body as unknown as Keys).clientId;
+        newClientSecret = (response.body as unknown as Keys).clientSecret;
 
-      expect(response.status).to.eq(200);
-      expect(Object.keys(body)).to.have.length(2);
-      expect(Object.keys(body)).to.include("clientId");
-      expect(Object.keys(body)).to.include("clientSecret");
-    });
+        expect(response.status).to.eq(200);
+        expect(Object.keys(body)).to.have.length(2);
+        expect(Object.keys(body)).to.include("clientId");
+        expect(Object.keys(body)).to.include("clientSecret");
+
+        return newClientId;
+      })
+      .then((newClientId) => {
+        cy.request({
```

Comment:
You're not returning this request, so how is that last then on 114 getting a response?

Reply:
Ah good call. Can't see clearly enough without having an ide